src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
-
+spring.jpa.hibernate.ddl-auto=none
+spring.datasource.url=jdbc:mysql://localhost:3306/userbase
```

Comment:
please also create a docker-compose.yml file for running this mysql instance locally

Reply:
Bump - add docker-compose.yml file